miniapps/meshing/mesh-explorer.cpp

```diff
@@ -221,8 +220,10 @@ int main (int argc, char *argv[])
            "x) Print sub-element stats\n"
            "f) Find physical point in reference space\n"
            "p) Generate a partitioning\n"
+           "o) Reorder elements\n"
```

Comment:
Besides `e) View elements` is there a better way to visualize the ordering?

Reply:
I don't know of a direct way. I was just using the new "1D split" partitioning to visualize the partitions. The "curve" plot in the PR description is from Octave.